marklogic-data-hub-central/ui/e2e/cypress/e2e/explore/valuesToIgnore.cy.tsx

```diff
@@ -0,0 +1,104 @@
+import LoginPage from "../../support/pages/login";
+import curatePage from "../../support/pages/curate";
+import graphExplore from "../../support/pages/graphExplore";
+import explorePage from "../../support/pages/explore";
+import runPage from "../../support/pages/run";
+import {rulesetSingleModal} from "../../support/components/matching";
+
+describe("Verify values to ignore feature", () => {
+
+  before(() => {
+    cy.clearAllLocalStorage();
+    cy.clearAllSessionStorage();
+    cy.loginAsTestUserWithRoles("hub-central-flow-writer", "hub-central-match-merge-writer", "hub-central-mapping-writer", "hub-central-load-writer").withRequest();
+    LoginPage.navigateToMainPage();
+  });
+
+  it("Should merge when values do not match", () => {
+    cy.visit("/tiles/run");
+    cy.waitForAsyncRequest();
+
+    runPage.toggleExpandFlow("testValuesToIgnore");
+    runPage.getRunFlowButton("testValuesToIgnore").click();
+    cy.uploadFile("input/valuesToIgnore/values-to-ignore1.json");
+    cy.uploadFile("input/valuesToIgnore/values-to-ignore2.json");
+    cy.uploadFile("input/valuesToIgnore/values-to-ignore3.json");
+    cy.uploadFile("input/valuesToIgnore/values-to-ignore4.json");
+    cy.waitForAsyncRequest();
+    cy.findByTestId("mergeForValuesToIgnore-success", {timeout: 12000}).should("be.visible");
+    runPage.explorerLink("mergeForValuesToIgnore").click();
+    cy.wait(13000);
```

Comment:
@aniamy 13000 is way too much wait can we try adding cy.waitForAsyncRequest(); and something like cy.get("#switch-view-table").should("be.visible", {timeout: 12000}).click({force: true}); instead and also please make sure we are declaring elements in the pages and not directly in the test like cy.get("#switch-view-table")

Reply:
Cleaned up the code. Thanks!